COMMUNITY-ROLES.md

```diff
@@ -138,21 +136,21 @@ using one of the template messages below as a base.
    go to https://github.com/orgs/tldr-pages/people
    and add them to the organization as a member.
 
-3. Open a PR moving their name to the "Organization members" section
+3. Open a PR adding their name to the "Organization members" section
    in [MAINTAINERS.md](MAINTAINERS.md#organization-members).
-   Make sure to include `Closes #<issue number>` in the PR description.
+   **Make sure to include `Closes #<issue number>` in the PR description.**
    The issue will then be automatically closed once the PR is merged.
 
 ### Adding new organization owners
 
-1. Open an issue with the following message template (edit it as appropriate):
+1. Open an issue with the following message template (editing as appropriate):
 
    ```
    Hi, @username! You've been an active tldr-pages organization member for over 6 months.
```

Comment:
I would like to rearrange these message templates a bit (mostly add newlines in certain places, e.g. after `Hi @username!`), because they're looking a bit messy (#5769).  What do you think, @waldyrious?

Reply:
That particular change makes sense to me. The others, I'll have to wait an see before I have an opinion :)

I would suggest holding those changes until the currently open conversations are resolved, though (in case you're planning to make them in this PR), to make the review more streamlined.